satellite/referral/offersweb/pages/index.html

```diff
@@ -0,0 +1,24 @@
+{{define "head"}}
+    <title>Referral Admin Portal</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="../static/css/style.css">
```

Comment:
We may be able to use the CSS code of the Kiev Team, to have the same look and feel. Having another UI with different UX experience feels wrong to me

Reply:
that makes sense

Reply:
upon review - I disagree with this. The satellite UI uses scoped css specific for each page on the satellite. In other words, they don't have a global css file. They have file specific for the login page, another file specific to the register page, etc..... I'd like to not use their styles and keep a new set of styles for the admin gui based on the mockups Atikh is making. @stefanbenten 

With that said I think a separate static folder for this service makes sense.

Reply:
Maybe you should collaborate with @tikhbana about this. I dont think it's a good idea to have x different UI flows and looks. Its just a recommendation tho.
We might want to unifi it in an additional PR then.

Reply:
I think you may be misinterpreting me @stefanbenten. Mockups for this are already in progress. How it looks is up to DJ and @tikhbana already. What I'm saying is the satellite UI does not use a global css file. It uses scoped css for each of it's different pages example : login.scss, register.scsss. Using the scoped css file for another page on another service doesn't make sense since those classes would not apply.